### ISSUE TITLE
Modal dialogs disappear in the background 

### ISSUE THREAD

**Issue by MoritzWeber0** [DSD-DBS/capella-dockerimages#317]
When a modal dialog / popup opens in Capella and the user clicks next to the window, the popups goes to the background and is hidden by the main Capella window. The main Capella window is unresponsive then. This is confusing to users and model dialogs should always stay in the foreground.

**Comment by jamilraichouni** [comment]
I prepared a script named `place_properties_window_on_top.py` and successfuly tested, if the following works in the `/tmp/supervisord/supervisord.xpra.conf`:

```ini
[program:place-properties-window-on-top]
command=python /path/to/place_properties_window_on_top.py
user=techuser
```
Question, where do we want to put the script?

The script itself is a service and will look like this:

```python
# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
# SPDX-License-Identifier: Apache-2.0
"""Script placing "Properties" windows on top of Capella's main window."""

import time

from Xlib import X, display


def _place_property_window_on_top(window):
    for child in window.query_tree().children:
        name = child.get_wm_name() if child.get_wm_name() else "Unknown"
        if name.lower().strip() == "properties":
            child.configure(stack_mode=X.PlaceOnTop)


def _main() -> None:
    while True:
        windows = display.Display().screen().root.query_tree().children
        if windows is None or not windows:
            # raise SystemExit(0)
            time.sleep(2)
            continue
        try:
            _ = [_place_property_window_on_top(w) for w in windows]
        except Exception:
            pass
        time.sleep(0.3)


if __name__ == "__main__":
    _main()
```

**Comment by MoritzWeber0** [comment]
> Question, where do we want to put the script?

My proposal would be to place it in `/opt`. That's where Capella lives. 

The `metrics` service was placed in `/home/techuser`, but I'd say that `/opt` is a more suitable location.

**Comment by jamilraichouni** [comment]
What about introducing a new dir `/opt/services`? This is our first but (IMHO) not last service.

**Comment by MoritzWeber0** [comment]
> What about introducing a new dir `/opt/services`? This is our first but (IMHO) not last service.

In theory, everything in `/opt` is just a service. Capella is also just a service that is started by supervisord. 
I'd put it to `/opt` directly. If the folder should get messy at some point (which I don't think will happen), we can still move it (without a breaking change).

**Comment by jamilraichouni** [comment]
I a peer-to-peer at work discussion between @MoritzWeber0 and me we decided to introduce a new dir named `/opt/services`.

I was short before proposing `/opt/startup` as parent dir, Reason is given in https://github.com/DSD-DBS/capella-dockerimages/blob/144f90c02b80d06391bb576b48ed4a762b4bd09e/capella/setup/README.md

But: This is dangerous, since a depending caller waits for all scripts in `/opt/startup` to be terminated and such a service in `setup` would end up in a kind of container deadlock.

**Comment by jamilraichouni** [comment]
> > Question, where do we want to put the script?
> 
> My proposal would be to place it in `/opt`. That's where Capella lives too.
> 
> The `metrics` service was placed in `/home/techuser`, but I'd say that `/opt` is a more suitable location.

We agreed on moving the `/home/techuser/.metrics.py`out of the `$HOME` and I suggest to rename it to
`/opt/services/metrics.py` which is not a hidden file anymore.

**Comment by jamilraichouni** [comment]
Storing this because it might become relevant when I open the PR:

<img width="896" alt="image" src="https://github.com/user-attachments/assets/559759cb-c7db-43b5-839f-a6874ab5aec6">

**Comment by jamilraichouni** [comment]
Solution, that works:

/etc/xdg/openbox/rc.xml

```xml
    <application type="normal">                                                                                                                                                                                                                                                                                                                    
      <fullscreen>yes</fullscreen>                                                                                                                                                                                                                                                                                                                 
    </application>  
```

**Comment by jamilraichouni** [comment]
There is a draft PR already https://github.com/DSD-DBS/capella-dockerimages/pull/328

**Comment by zusorio** [comment]
Fixed by #334
